buildtools/pom.xml

```diff
@@ -29,7 +29,7 @@
 
   <groupId>org.apache.mahout</groupId>
   <artifactId>mahout-buildtools</artifactId>
-  <version>0.11.1-SNAPSHOT</version>
+  <version>0.12.0-SNAPSHOT</version>
```

Comment:
Given that this is a Spark upgrade release + Bug fixes, lets keep the release at 0.11.1-Snapshot